includes/wikia/controllers/HeliosHelperController.class.php

```diff
@@ -52,8 +50,7 @@ public function checkAntiSpoof()
 	 *
 	 * @see extensions/AntiSpoof
 	 */
-	public function updateAntiSpoof()
-	{
+	public function updateAntiSpoof() {
 		$this->response->setFormat( 'json' );
 		$this->response->setCacheValidity( \WikiaResponse::CACHE_DISABLED );
 		$this->response->setVal( 'success', false );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "success" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)